### PR TITLE
Sync graph

### PR DESCRIPTION
**Pull request by johnedquinn**
Add full functionality for grabbing file. The only thing that will need to be updated is deciding which file to run (waiting on Pam's end). But, it is fully functional beyond integrating for daily run.

### CODE REVIEW

File: syncGraph.py [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+    '''
+    @desc   Connects to mysql
+    --
+    @param  config  mysql.connector config object
+    '''
+    try:
+        return mysql.connector.connect(**config)
+    except mysql.connector.Error:
+        raise
+        return None
```
Comment (danjr26) [inline syncGraph.py]: Unreachable?

File: syncGraph.py [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+    return posts, edges
+
+
+def getSubimageWeightsMeta(edges, source, target):
+    '''
+    @desc  Returns subimage weight (float, else None) and metadata (string, else None)
+    '''
+    sub_edges = edges[source][target].get('subimage', [])
+    labels = []
+    sw = 0 if len(sub_edges) > 0 else None
```
Comment (danjr26) [inline syncGraph.py:108]: I might suggest more descriptive variable names ('weight' and 'metadata', maybe?)

File: syncGraph.py [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+        'sm': sm,
+        'ow': ow,
+        'om': om
+    }
+
+    # Run Query
+    try:
+        cursor.execute(sql, args)  
+    except mysql.connector.Error:
+        raise
```
Comment (danjr26) [inline syncGraph.py:209]: What is the purpose of the except/raise?

Comment (johnedquinn) [reply syncGraph.py:209]: Re-raising the exception so that line 246 can handle it appropriately.

File: syncGraph.py (hunk 2 above)
Comment (johnedquinn) [reply syncGraph.py:108]: Note taken. Will update.

File: syncGraph.py (hunk 1 above)
Comment (johnedquinn) [reply syncGraph.py]: Yeah, missed that. Will provide error message and exit. (cause it's only called once at beginning of main).

**Review by danjr26 — APPROVED** [review]

**Pull request merged**